website/docs/patterns/table-multi-select/partials/interaction-concepts/interaction-concepts.md

```diff
@@ -57,7 +57,7 @@ Filtering a data set is a reductive action that removes results from being displ
 Consider these topics as conversation starters to help make an informed decision about the user experience. The answers may vary depending on product architecture and broader application strategy.
 !!!
 
-- In a Multi-select pattern, should the [total count](/patterns/multi-select-patterns?tab=guidlines#total-count) change to reflect a reduction in the number of visible results?
+- In a Multi-select pattern, should the [total count](/patterns/table-multi-select?tab=guidlines#total-count) change to reflect a reduction in the number of visible results?
```

Comment:
Same question here, but also "guidelines" was mispelled.

```suggestion
- In a Multi-select pattern, should the [total count](/patterns/table-multi-select#total-count) change to reflect a reduction in the number of visible results?
```

Reply:
good catch! Yup, just totally removed it from link so anchor scrolling is fixed as well.